### PR TITLE
OrchardCore 1.4.0

### PR DESCRIPTION
**Pull request by agriffard**
https://github.com/OrchardCMS/OrchardCore/releases/tag/v1.4.0

### CODE REVIEW

File: OrchardCore.Commerce/Views/AddressField_Edit.cshtml [hunk 1]
```diff
@@ -4,74 +4,74 @@
 
 @{ var settings = Model.PartFieldDefinition.GetSettings<AddressPartFieldSettings>(); }
 <div class="row">
-    <div class="form-group col-md-12">
+    <div class="mb-3 col-md-12">
```
Comment (barthamark) [inline OrchardCore.Commerce/Views/AddressField_Edit.cshtml:7]: What's the reason for removing `form-group` classes and adding margin utility classes instead?

Comment (Skrypt) [reply OrchardCore.Commerce/Views/AddressField_Edit.cshtml:7]: form-group has been removed from BS5 and it was basically only adding a bottom margin.

**Comment by agriffard** [comment]
Error in UI tests:
`Execute behavior ClicksUsingClickMethodAttribute against "Items" unordered list's "Title == "Test page"" item's "View" link (5.022s) >> OpenQA.Selenium.NoSuchElementException: Unable to locate "Items" unordered list's "Title == "Test page"" item element:...`

**Comment by DAud-IcI** [comment]
@agriffard Try updating _Lombiq.Tests.UI_ packages [here](https://github.com/OrchardCMS/OrchardCore.Commerce/blob/303be811f78c282f562c70a27ca161f6fa4fa144/test/OrchardCore.Commerce.Tests.UI/OrchardCore.Commerce.Tests.UI.csproj#L11-L13) to the [latest release](https://github.com/Lombiq/UI-Testing-Toolbox/releases).
It needs to be at least [v3.1.0](https://github.com/Lombiq/UI-Testing-Toolbox/releases/tag/v3.1.0#:~:text=upgrade%20to%20orchard%20core%201.4) for OC 1.4 compatibility, but we might as well update it all the way to [v3.4.0](https://github.com/Lombiq/UI-Testing-Toolbox/releases/tag/v3.4.0).

**Comment by agriffard** [comment]
> @agriffard Try updating _Lombiq.Tests.UI_ packages [here](https://github.com/OrchardCMS/OrchardCore.Commerce/blob/303be811f78c282f562c70a27ca161f6fa4fa144/test/OrchardCore.Commerce.Tests.UI/OrchardCore.Commerce.Tests.UI.csproj#L11-L13) to the [latest release](https://github.com/Lombiq/UI-Testing-Toolbox/releases). It needs to be at least [v3.1.0](https://github.com/Lombiq/UI-Testing-Toolbox/releases/tag/v3.1.0#:~:text=upgrade%20to%20orchard%20core%201.4) for OC 1.4 compatibility, but we might as well update it all the way to [v3.4.0](https://github.com/Lombiq/UI-Testing-Toolbox/releases/tag/v3.4.0).

Done. Thank you @DAud-IcI.

**Comment by DAud-IcI** [comment]
Please merge from `main`.

File: src/Modules/OrchardCore.Commerce/Views/PricePart.Edit.cshtml [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
-    <div class="form-group" asp-validation-class-for="Price">
+    <div class="mb-3" asp-validation-class-for="Price">
         <label asp-for="Price">@T["Price"] </label>
         <div class="input-group">
             <input asp-for="PriceValue" class="form-control text-muted" />
             <div class="input-group-append">
-                <select asp-for="PriceCurrency"
+                <select asp-for="PriceCurrency" class="form-select"
                         asp-items="@(new SelectList(Model.Currencies, "CurrencyIsoCode", "CurrencyIsoCode"))"></select>
```
Comment (DAud-IcI) [inline src/Modules/OrchardCore.Commerce/Views/PricePart.Edit.cshtml:9]: Please break each subsequent attribute into a new line if they don't fit in one line.
```suggestion
                <select asp-for="PriceCurrency"
                        class="form-select"
                        asp-items="@(new SelectList(Model.Currencies, "CurrencyIsoCode", "CurrencyIsoCode"))"></select>
```

File: src/Modules/OrchardCore.Commerce/Views/PricePartSettings.Edit.cshtml [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 
-<fieldset class="form-group" asp-validation-class-for="PricePartSettings">
+<fieldset class="mb-3" asp-validation-class-for="PricePartSettings">
     <label asp-for="CurrencySelectionMode">@T["Currency selection mode"]</label>
 
     <div class="input-group">
         <div class="input-group-append">
-            <select asp-for="CurrencySelectionMode"
+            <select asp-for="CurrencySelectionMode" class="form-select"
                     asp-items="@(new SelectList(Model.CurrencySelectionModes, "Text", "Value"))"></select>
```
Comment (DAud-IcI) [inline src/Modules/OrchardCore.Commerce/Views/PricePartSettings.Edit.cshtml:9]:
```suggestion
            <select asp-for="CurrencySelectionMode"
                    class="form-select"
                    asp-items="@(new SelectList(Model.CurrencySelectionModes, "Text", "Value"))"></select>
```

File: src/Modules/OrchardCore.Commerce/Views/PricePartSettings.Edit.cshtml [hunk 4]
```diff
@@ -15,7 +15,7 @@
     <fieldset class="input-group" id="@Html.IdFor(model => model.SpecificCurrencyIsoCode)-Group">
         <div class="input-group">
             <div class="input-group-append">
-                <select asp-for="SpecificCurrencyIsoCode"
+                <select asp-for="SpecificCurrencyIsoCode" class="form-select"
                         asp-items="@(new SelectList(Model.Currencies, "Text", "Value"))"></select>
```
Comment (DAud-IcI) [inline src/Modules/OrchardCore.Commerce/Views/PricePartSettings.Edit.cshtml:19]:
```suggestion
                <select asp-for="SpecificCurrencyIsoCode"
                        class="form-select"
                        asp-items="@(new SelectList(Model.Currencies, "Text", "Value"))"></select>
```

File: src/Modules/OrchardCore.Commerce/Views/PriceVariantsPart.Edit.cshtml [hunk 5]
```diff
@@ -6,12 +6,12 @@
...
         <div class="col-md-3">
-            <div class="form-group" asp-validation-class-for="VariantsValues[variantKey]">
+            <div class="mb-3" asp-validation-class-for="VariantsValues[variantKey]">
                 <label asp-for="VariantsValues[variantKey]">@variantKey @T["Price"] </label>
                 <div class="input-group">
                     <input asp-for="VariantsValues[variantKey]" class="form-control text-muted variant-price-value" />
                     <div class="input-group-append">
-                        <select asp-for="VariantsCurrencies[variantKey]"
+                        <select asp-for="VariantsCurrencies[variantKey]" class="form-select"
                                 asp-items="@(new SelectList(Model.Currencies, "CurrencyIsoCode", "CurrencyIsoCode"))"></select>
```
Comment (DAud-IcI) [inline src/Modules/OrchardCore.Commerce/Views/PriceVariantsPart.Edit.cshtml:15]:
```suggestion
                        <select asp-for="VariantsCurrencies[variantKey]"
                                class="form-select"
                                asp-items="@(new SelectList(Model.Currencies, "CurrencyIsoCode", "CurrencyIsoCode"))"></select>
```

**Review by DAud-IcI — CHANGES_REQUESTED** [review]

**Pull request merged**